pkg/endpoint/policy.go

```diff
@@ -78,8 +78,12 @@ func (e *Endpoint) updateNetworkPolicy(proxyWaitGroup *completion.WaitGroup) (re
 		return nil, nil
 	}
 
+	if e.proxy == nil {
```

Comment:
Hmm... this is not necessarily true.. check https://github.com/cilium/cilium/issues/9100 or at least comment somewhere this is being added

Reply:
I am preserving existing behavior here: the call to `e.owner.UpdateNetworkPolicy` has now been removed. The logic in the `Owner`'s `UpdateNetworkPolicy` was the following:

```
// UpdateNetworkPolicy adds or updates a network policy in the set		
 // published to L7 proxies.		
 func (d *Daemon) UpdateNetworkPolicy(e regeneration.EndpointUpdater, policy *policy.L4Policy,		
 	proxyWaitGroup *completion.WaitGroup) (error, revert.RevertFunc) {		
 	if d.l7Proxy == nil {		
 		return fmt.Errorf("can't update network policy, proxy disabled"), nil		
 	}		
 	err, revertFunc := d.l7Proxy.UpdateNetworkPolicy(e, policy, e.GetIngressPolicyEnabledLocked(),		
 		e.GetEgressPolicyEnabledLocked(), proxyWaitGroup)		
 	return err, revert.RevertFunc(revertFunc)		
 }
```